core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-toolbar/edit-ema-toolbar.component.html

```diff
@@ -1,9 +1,11 @@
-<p-menu #menu [model]="items" [popup]="true"></p-menu>
-<button
-    class="p-button-sm p-button-text"
-    [attr.aria-label]="label"
-    [label]="label"
-    (click)="menu.toggle($event)"
-    pButton
-    type="button"
-    icon="pi pi-globe"></button>
+<h4 data-testId="page-title">{{ pageTitle }}</h4>
+
+<div class="edit-ema-toolbar-actions">
+    <div class="edit-ema-toolbar-actions__buttons">
+        <ng-content select="[left]"></ng-content>
+    </div>
+
+    <div class="edit-ema-toolbar-actions__buttons">
+        <ng-content select="[right]"></ng-content>
+    </div>
+</div>
```

Comment:
https://primeng.org/toolbar I don't know how this looks like right now in our global styles but I can tell you that all our toolbars should look the same.